CMakeLists.txt

```diff
@@ -1244,17 +1244,7 @@ install(
 
 
 # Font files
-if(UNIX AND NOT APPLE)
-  # Ubuntu and OpenSans are already installed by package manager
-  install(
-    DIRECTORY
-      "${CMAKE_CURRENT_SOURCE_DIR}/res/fonts"
-    DESTINATION
-      "${MIXXX_INSTALL_DATADIR}"
-    PATTERN "OpenSans*" EXCLUDE
-    PATTERN "Ubuntu*" EXCLUDE
-  )
-else()
+if(NOT UNIX OR NOT APPLE)
```

Comment:
`if(WIN32)`?

Reply:
Good catch, it should be `if(NOT LINUX)` but that doesn't exist. So we use `if(NOT UNIX Or APPLE)`.